src/OpenTelemetry.Extensions.Enrichment/OpenTelemetry.Extensions.Enrichment.csproj

```diff
@@ -1,7 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(NetFrameworkMinimumSupportedVersion);$(NetStandardMinimumSupportedVersion)</TargetFrameworks>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>net7.0;net6.0;</TargetFrameworks>
```

Comment:
For non Windows OS we are going to support NET 6 & 7, and for Windows - .NET Framework, .NET Standard, .NET 6 & 7, right? I'm asking just to confirm.

Reply:
Correct, but I think .NET Standard shouldn't be only for windows, I'll update it.